src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -117,7 +117,21 @@ private CloudSdk(Path sdkPath, String appCommandMetricsEnvironment,
    * @throws CloudSdkOutOfDateException when the installed Cloud SDK is too old 
    */
   public void runAppCommand(List<String> args) throws ProcessRunnerException {
-    runGcloudCommand(args, "app");
+    runGcloudCommand(args, null, "app");
```

Comment:
nit: How do you feel about adding the field names inside comment blocks for the null and string literal arguments?